_docs/api/JsonSchema.Net.Generation/JsonExcludeAttribute.md

```diff
@@ -22,7 +22,7 @@ Indicates that the property should be excluded from generation.
 
 This attribute functions exactly the same as the **System.Text.Json.Serialization.JsonIgnoreAttribute**.  It
 is included separately to support the case where the model should be serialized with
-a property but schema generation should ignore it.
+a property or an enum member but schema generation should ignore it.
```

Comment:
There's no need to update thes /api docs.  These will be rebuilt automatically from the source after the libs merge to master.

You'll need to back out this change and force push.  Otherwise, there will be merge conflict (and you'll have to do it anyway).

Reply:
Ah, but I didn't update the XML doc comment on `JsonExclude`. Need to do that, then!

Reply:
> TBH, I didn't carefully read the verbiage saying it only applies to properties (otherwise I wouldn't have expected it to work on enums).

I still think it's an improvement.

Reply:
Sorry about that. PR: https://github.com/json-everything/json-everything/pull/801

Reply:
> I still think it's an improvement.

Yeah, that's what I was saying, I was just projecting my assumptions, but the documentation clearly stated the intent